elements/pfe-navigation/src/pfe-navigation-item.js

```diff
@@ -0,0 +1,429 @@
+      return this.getAttribute("pfe-icon");
+    }
+  
+    get nested() {
+      return this.hasAttribute("is_nested");
+    }
+  
+    set nested(isNested) {
+      isNested = Boolean(isNested);
+  
+      if (isNested) {
+        this.setAttribute("is_nested", "");
+      } else {
+        this.removeAttribute("is_nested");
+      }
+    }
+  
+    get expanded() {
+      return this.classList.contains("expanded");
+    }
+  
+    set expanded(isExpanded) {
+      isExpanded = Boolean(isExpanded);
+  
+      if (isExpanded) {
+        this.classList.add("expanded");
+  
+        if (this.iconName === "web-mobile-menu") {
+          if (this._icon) this._icon.setAttribute("icon", "web-plus");
+        }
+
+        if (this._trigger) {
+          this._trigger.setAttribute("aria-expanded", true);
+        }
+  
+        if (this.tray) {
+          this.tray.removeAttribute("hidden");
+        }
+  
+        if (this._tray) {
+          this._tray.setAttribute("aria-expanded", true);
+        }
+      } else {
+        this.classList.remove("expanded");
+  
+        if (this.iconName === "web-mobile-menu") {
+          if (this._icon) this._icon.setAttribute("icon", "web-mobile-menu");
+        }
+  
+        if (this._trigger) {
+          this._trigger.setAttribute("aria-expanded", false);
+        }
+  
+        if (this.tray) {
+          this.tray.setAttribute("hidden", "");
+        }
+  
+        if (this._tray) {
+          this._tray.setAttribute("aria-expanded", false);
+        }
+      }
+    }
+  
+    get visible() {
+      return !this.hasAttribute("hidden");
+    }
+  
+    set visible(isVisible) {
+      isVisible = Boolean(isVisible);
+  
+      if (isVisible) {
+        this.removeAttribute("hidden");
+      } else {
+        this.setAttribute("hidden", "");
+      }
+    }
+  
+    open(event) {
+      if (event) event.preventDefault();
+  
+      // Close the other active item(s) unless it's this item's parent
+      if (this.navigationWrapper) {
+        this.navigationWrapper._activeNavigationItems = this.navigationWrapper._activeNavigationItems.filter(item => {
+          let stayOpen = item === this.parent;
+          if (!stayOpen) item.close();
+          return stayOpen;
+        });
+  
+        // Open that item and add it to the active array
+        this.navigationWrapper._activeNavigationItems.push(this);
+  
+        this.expanded = true;
+        this.navigationWrapper.overlay = true;
+      }
+  
+      // Dispatch the event
+      this.dispatchEvent(
+        new CustomEvent(`${this.tag}:open`, {
+          detail: {},
+          bubbles: true,
+          composed: true
+        })
+      );
+    }
+  
+    close(event) {
+      if (event) event.preventDefault();
+  
+      // Close the children elements
+      this.navigationWrapper._activeNavigationItems = this.navigationWrapper._activeNavigationItems.filter(item => {
+        let close = this.nestedItems && this.nestedItems.includes(item);
```

Comment:
`.includes` is not available in IE11

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
I'll add the polyfill